### PR TITLE
SDIT-1482 Mapping endpoints for Locations

### PR DESCRIPTION
**Pull request by steverendell**
Switch to string type for DPS PK (really a UUID)

### CODE REVIEW

File: src/main/resources/db/migration/V1_35__location.sql [hunk 1]
```diff
@@ -0,0 +1,13 @@
+drop table location_mapping;
+
+create table location_mapping (
```
Comment (karenmillermoj) [inline src/main/resources/db/migration/V1_35__location.sql:3]: I was looking at this for incidents aswell, wondering if we should have an index on the nomis id too?  None of the other tables do, but our main search is usually on the nomis value...what do you think?

**Review by karenmillermoj — DISMISSED** [review]
lgtm

File: src/main/resources/db/migration/V1_35__location.sql (hunk 1 above)
Comment (steverendell) [reply src/main/resources/db/migration/V1_35__location.sql:3]: v good point yes, deff needs an index! Will add

**Review by karenmillermoj — APPROVED** [review]
lgtm

**Pull request merged**